monky.el

```diff
@@ -2882,6 +2890,19 @@ With a non numeric prefix ARG, show all entries"
       (error "Nothing staged")
     (monky-pop-to-log-edit 'commit)))
 
+(defun monky-commit-amend ()
+  "Amends previous commit.
+Brings up a buffer to allow editing of commit message."
+  (interactive)
+  (if (not (or monky-staged-files (monky-merge-p)))
+      (error "Nothing staged")
+	;; get last commit message
+	(with-current-buffer (get-buffer-create monky-log-edit-buffer-name)
+	  (monky-hg-insert
+	   (list "log"
+			 "--template" "{desc}" "-r" "-1")))
```

Comment:
I think "-1" should be "." here, for when you haven't checked out tip.  (This is wrong elsewhere in the file, too.)